sdk/search/azure-search-documents/tests/async_tests/test_batching_client_async.py

```diff
@@ -17,37 +17,34 @@
 
 class TestSearchBatchingClientAsync(object):
     async def test_search_index_document_batching_client_kwargs(self):
-        client = SearchIndexDocumentBatchingClient("endpoint", "index name", CREDENTIAL, window=100)
-
-        assert client.batch_size == 1000
-        assert client._window == 100
-        assert client._auto_flush
-        await client.close()
+        async with SearchIndexDocumentBatchingClient("endpoint", "index name", CREDENTIAL, window=100) as client:
```

Comment:
It doesn't look like you have tests for the new kwargs (i.e. `error_callback`, `progress_callback` etc). can you add tests for these?